README.md

```diff
@@ -1,12 +1,12 @@
+|[16.0 Preview 1](docs/repo/roadmap.md)|[dev16.0.x](https://github.com/dotnet/project-system/tree/dev16.0.x)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/project-system/unit-tests.debug.batched?branchName=dev16.0.x)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=152&branchName=dev16.0.x) |[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/project-system/unit-tests.release.batched?branchName=dev16.0.x)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=153&branchName=dev16.0.x)| [![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/project-system/unit-tests.debug.spainish.batched?branchName=dev16.0.x)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=157&branchName=dev16.0.x) |[![Build Status](https://ci.dot.net/job/dotnet_project-system/job/dev16.0.x/job/windows_integration_debug/badge/icon)](https://ci.dot.net/job/dotnet_project-system/job/dev16.0.x/job/windows_integration_debug/) |[![codecov](https://codecov.io/gh/dotnet/project-system/branch/dev16.0.x/graph/badge.svg)](https://codecov.io/gh/dotnet/project-system)
+|[16.0 Preview 2](docs/repo/roadmap.md)|[master](https://github.com/dotnet/project-system/tree/master)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/project-system/unit-tests.debug.batched?branchName=master)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=152&branchName=master)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/project-system/unit-tests.release.batched?branchName=master)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=153&branchName=master)| [![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/project-system/unit-tests.debug.spainish.batched?branchName=master)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=157&branchName=master) |[![Build Status](https://ci.dot.net/job/dotnet_project-system/job/master/job/windows_integration_debug/badge/icon)](https://ci.dot.net/job/dotnet_project-system/job/master/job/windows_integration_debug/) |[![codecov](https://codecov.io/gh/dotnet/project-system/branch/master/graph/badge.svg)](https://codecov.io/gh/dotnet/project-system)
```

Comment:
Not sure if it's an artifact of github's rich diff, but this is showing both builds?

<img width="225" alt="screen shot 2018-10-09 at 1 40 05 pm" src="https://user-images.githubusercontent.com/312835/46697307-2ede8480-cbc9-11e8-9e76-20831f9dc786.png">

Reply:
yeah, its trying to show you the diff,  go [here](https://github.com/dotnet/project-system/blob/9180ccae5d4eec89509cbf3f9c67df787f7ac075/README.md) to view the actual file